scheduletemplates/models.py

```diff
@@ -71,18 +71,16 @@ class Meta:
 
     @property
     def duration(self):
-        today = datetime.today()
-        days = timedelta(days=self.days)
+        today = timezone.now().date()
         start = datetime.combine(today, self.starting_time)
-        end = datetime.combine(today + days, self.ending_time)
+        end = datetime.combine(today + timedelta(days=self.days), self.ending_time)
         duration = end - start
         return duration
 
     @property
     def localized_display_ending_time(self):
         days = self.days if self.ending_time > time.min else 0
-        days_fmt = ungettext_lazy(u'the next day',
-                                  u'after {number_of_days} days', days)
+        days_fmt = _(u'after {number_of_days} days'.format(number_of_days=days))
```

Comment:
Why trashing pluralized translation?

Reply:
ugettext_lazy vs ungettext_lazy changed its signiture and I felt that this is not a big miss ;-)